### ISSUE TITLE
Not able to forfeit a block

### ISSUE THREAD

**Issue by vineet4183** [simonpacis/AmazonFlexUnlimitedSUPERCHARGED#2]
Hi,
I have not been able to forfeit a block. tried multiple combinations.
Anyone able to do it successfully?

**Comment by cpietrobelli** [comment]
Hi
Can you explain the question a bit better?
- Forfeit a block you do it through the amazon flex app
- If the question is that you can capture blocks with the script, the answer is yes.

**Comment by vineet4183** [comment]
Hi,
My question is how can we forfeit a block with the script?

Regards,
Vineet Chaudhary
________________________________

**Comment by cpietrobelli** [comment]
No, the script does not have that option, you have to do it through the Amazon flex app

**Comment by vineet4183** [comment]
Yeah, I understand that this script does not have an option to forfeit a block. But, there are bot sites out there that auto forfeit the block within a certain period of time.
So it is possible to do. Can this script replicate the same?

Regards,
Vineet Chaudhary
________________________________

**Comment by cpietrobelli** [comment]
If you have to ask programmers to add this function

**Comment by vineet4183** [comment]
That is what I'm asking

Regards,
Vineet Chaudhary
________________________________
